netbench/cdk/src/main/java/com/aws/NetbenchAutoApp.java

```diff
@@ -88,8 +88,11 @@ public static void main(final String[] args) {
             .instanceType(ec2InstanceType)
             .ecrUri(serverEcrUri)
             .scenario(scenarioFile)
+            .serverRegion(serverRegion)
             .build());
 
+        serverEcsStack.addDependency(vpcStack);
```

Comment:
is this call necessary? The cdk should be able to deduce the dependency in this scenario

Reply:
I don't think so, I added it just to make the stack dependencies explicit, but I can remove it if you think that's unnecessary

Reply:
I would recommend removing it. As the cdk expands, we might add additional stack with more dependencies and this can cause some errors